### ISSUE TITLE
wiki out-of-date?

### ISSUE THREAD

**Issue by thautwarm** [xoofx/CppAst.NET#62]
I'd appreciate it for the project's informative and categorical documentations.
Unfortunately,  some of the docs seem inaccurate.

[CppAst.NET type-system](https://github.com/xoofx/CppAst.NET/blob/main/doc/readme.md#type-system) gives a clarified description of the possible variants of `CppType`s, but `CppTemplateParameterType` is not included.

I'd ask if the wiki is up-to-date, or anything else I'd take care if I use this project?

**Comment by xoofx** [comment]
Yes, the doc is not up-to-date, some folks made PRs to bring support for template parameters.
